src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -3,18 +3,23 @@ class Webui::CommentsController < Webui::WebuiController
   before_action :find_commentable, only: :create
 
   def create
-    comment = @commented.comments.new(permitted_params)
-    User.session!.comments << comment
-    @commentable = comment.commentable
+    if @commented.nil?
+      flash.now[:error] = 'Failed to create comment: The object the comment refers to does not exist anymore.'
```

Comment:
`object` is not something your customers will understand.

Reply:
Yeah, I'm still working on that messages. I don't like it either

Reply:
As discussed with @danidoni, we can tell for which kind of object the comment was for. This is passed in the params (`params[:commentable_type]`) when the form is submitted.